### DISCUSSION TITLE
brcharts logo

### DISCUSSION THREAD

**Discussion by Lovemore-Gakava** [BR-Visualization/brcharts#71] [Ideas]
@chen-chen-stat @MikeColopy @aemabe - space to refine and select the logo for brcharts.

**Comment by aemabe** [comment]
![image](https://github.com/user-attachments/assets/d0834ad9-1002-4a60-8374-2a2da67fc3e2)

Here's an option for a Hex logo utilizing an online image of scales. The original image was touched up with a combination of Photoshop and R to create a Hex frame.

**Reply by aemabe** [reply]
![hexsticker2(1)](https://github.com/user-attachments/assets/30c55216-62f1-44a7-b18d-785fcc6883ad)
![hexsticker25(1)](https://github.com/user-attachments/assets/641dcf46-4d3a-4402-b76a-3bc02082f830)

Here are a few more options for the Hex logo. The sides for benefit and risk are switched around and the border is a gradient to reflect the colors within. The two variations differ by their font.

**Reply by aemabe** [reply]
![image](https://github.com/user-attachments/assets/0579d3b0-0ed6-47db-a012-aa7c1ce5fb75)

**Reply by aemabe** [reply]
![image](https://github.com/user-attachments/assets/dec21bad-d4e7-49d2-8d53-d13c8aed36cb)
![image](https://github.com/user-attachments/assets/28364abe-ba45-4b99-a1f5-36950b781586)

**Reply by aemabe** [reply]
![hexsticker27(1)](https://github.com/user-attachments/assets/fb07ac03-d1d3-4c6b-af1f-7e8cd08c7d73)

**Comment by MikeColopy** [comment]
Of the four choices, this one is my favorite.  [image: image.png]
Black font and a thin black border. The green and red backgrounds help
differentiate between benefit and risk.
[image: image.png]

Very appealing and professional looking.
Kind regards,
Mike

**Reply by Lovemore-Gakava** [reply]
Thanks @MikeColopy - team we have a winner:
![selected](https://github.com/user-attachments/assets/66b7b566-8c69-4678-aed6-b2417abf3a08)
